src/layouts/ha-init-page.ts

```diff
@@ -7,13 +7,20 @@ class HaInitPage extends LitElement {
 
   @state() showProgressIndicator = false;
 
+  @state() retryInSeconds = 60;
+
   private _showProgressIndicatorTimeout;
 
+  private _retryInterval;
```

Comment:
```suggestion
  private _retryInterval: number;
```

Reply:
This breaks under modern TS - but updated to use explicit types.